src/raftstore/store/worker/raftlog_gc.rs

```diff
@@ -51,9 +56,15 @@ quick_error! {
     }
 }
 
-pub struct Runner;
+pub struct Runner {
+    ch: Option<Sender<TaskRes>>,
```

Comment:
I think we can send the task, wait a little time and then check the rocksdb data directly, not the sender.